src/libsystemd/sd-event/sd-event.c

```diff
@@ -1494,6 +1499,22 @@ _public_ int sd_event_source_set_io_fd(sd_event_source *s, int fd) {
         return 0;
 }
 
+_public_ int sd_event_source_get_io_fd_close(sd_event_source *s, bool *close) {
```

Comment:
so far we used "bool" only internally, and exposed booleans in public APIs as "int", in order not to impose C99 boolean semantics on our users (whether that really made sense is a different question) — let's follow this here too, i.e. change bool → int in this argument here (but keep it internally in the struct a proper bool please)

Reply:
also, let's shorten the name by one word, please, let's drop the "fd" bit, I think "sd_event_source_get_io_close()" is explanatory enough

Reply:
How about sd_event_source_[gs]et_io_fd_own()? This resolves the name collision and shortens the function name while being pretty obvious.

Reply:
sounds good by me!